tests/unit/consensus/test_base.py

```diff
@@ -1,6 +1,9 @@
 from collections import defaultdict
```

Comment:
@kstroobants can we abstract the boilerplate code on these tests to another file so it is clear what we are testing?

Reply:
Done in #43da57eda8aad30fb3c2144d5930e1952e986d66